### PR TITLE
Fix broken HTML and XML highlighting

### PR DESCRIPTION
**Pull request by datakurre**
Added https://stackoverflow.com/a/5499821 to fix issue where ```html resulted in output without tags and ```xml resulted in output with broken tags

### CODE REVIEW

File: highlightCode.js [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
+   '&': '&amp;',
+   '<': '&lt;',
+   '>': '&gt;'
+};
+
+const replaceTag = function(tag) {
+   return tagsToReplace[tag] || tag;
+}
+
+const safe_tags_replace = function(str) {
```
Comment (MaxDesiatov) [inline highlightCode.js]: could you please make the naming consistent? The rest of the declarations use camelCase, while this one switches to snake case for some reason.

File: highlightCode.js [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
+    return str.replace(/[&<>]/g, replaceTag);
```
Comment (MaxDesiatov) [inline highlightCode.js]: this probably could avoid use of `replaceTag` function and corresponding object with a regex group substitution?

**Review by MaxDesiatov — CHANGES_REQUESTED** [review]
Thanks for catching the error, I have a few comments about styling and the regex use.

File: highlightCode.js (hunk 1 above)
Comment (datakurre) [reply highlightCode.js]: Thanks. I renamed the snake case function (originating from stack overflow) to camel case and renamed the functions to be more descriptive.

File: highlightCode.js (hunk 2 above)
Comment (datakurre) [reply highlightCode.js]: But bI wonder if that really makes the code any more clear? The current version is verbose, but quite obvious.

**Review by MaxDesiatov — APPROVED** [review]
LGTM, thanks 👍

**Pull request merged**